packages/controller/main.js

```diff
@@ -5290,6 +5298,10 @@ function stop(force, callback) {
         reportInterval = null;
     }
 
+    if (isStopping) {
+        return;
```

Comment:
maybe also log to see if it is the actual issue

Reply:
From logs can only be that way because there is no other option that the callback from stopInstances is called twice